skranger/ensemble/ranger_forest_classifier.py

```diff
@@ -295,3 +297,28 @@ def predict_log_proba(self, X):
         """
         proba = self.predict_proba(X)
         return np.log(proba)
+
+    def get_importance_pvalues(self):
+        """Return p-values for variable importance using the fast method from Janitza et al. (2016)
+        """
+
+        if self.importance != "impurity_corrected":
+            raise Exception("P-values can only be calculated for impurity corrected importance values")
```

Comment:
Let's raise a ValueError here, indicating that one of the initialization parameters is invalid for this method.

Reply:
We can be more explicit with the message here saying:
```
p-values can only be calculated with importance parameter set to `impurity_corrected`
```
I'm looking at this code for reference: https://github.com/imbs-hl/ranger/blob/e8b05f47892bb4968c4e6057f68b35bcd0b52225/R/importance.R